integration/integration_tests.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+# Basic binary compatibility test script for replace_landsurface.
+# See INTEGRATION_README.md for details on test usage, data and options.
+
+TEST_DATA_DIR=/g/data/vk83/testing/data/replace_landsurface/integration_tests
+INPUT_DIR=$TEST_DATA_DIR/input_data
+OUTPUT_DIR=$TEST_DATA_DIR/expected_outputs
+DRIVING_DATA_DIR=$TEST_DATA_DIR/driving_data
+CLEAN_OUTPUT=true
+
```

Comment:
```suggestion
INPUT_DIR=${TEST_DATA_DIR}/input_data
OUTPUT_DIR=${TEST_DATA_DIR}/expected_outputs
DRIVING_DATA_DIR=${TEST_DATA_DIR}/driving_data
CLEAN_OUTPUT=true
```
It is good practice to enclose shell variables in curly braces, `{}`, especially when concatenating strings. It won't have a material in effect here, but it's good practice to avoid strange errors

https://stackoverflow.com/a/8748880

Reply:
I usually don't use curly braces when it's not necessary (like in this case). 
But it's true, some nasty unexpected errors could always happen and putting them won't hurt, so I'm gonna add them.